src/fez.cr

```diff
@@ -3,25 +3,23 @@ require "option_parser"
 require "ecr/macros"
 
 OptionParser.parse! do |parser|
-  parser.banner = "Usage: fez [arguments]"
+  parser.banner = "Usage: fez [name] [--] [arguments]"
```

Comment:
I'm thinking maybe this should look like `Usage: fez NAME [--] [ARGS]`.  This would show that name is required.

Taking a look at rails:
```
[20:01PM] namechk (master)$ rails -h
Usage: rails COMMAND [ARGS]
```